ToolbarActionButton.tsx

```diff
@@ -1,26 +1,58 @@
 import { Platform, Pressable, StyleSheet, Text } from "react-native";
 import React from "react";
+import { PlatformPressable } from "@react-navigation/elements";
 
 export const ToolbarActionButton = ({
   label,
   onPress,
 }: ToolbarActionButtonProps) => {
-  const formattedLabel = Platform.OS === "android" ? label.toUpperCase() : label;
-
   return (
-    <Pressable android_ripple={{ borderless: true }} onPress={onPress}>
-      <Text style={styles.textStyle}>{formattedLabel}</Text>
-    </Pressable>
+    <PlatformPressable
+      style={styles.extraEdgeMarginOnRight}
+      android_ripple={rippleConfig}
+      onPress={onPress}
+    >
+      <Text style={styles.text}>{label}</Text>
+    </PlatformPressable>
   );
 };
+
 const styles = StyleSheet.create({
-  textStyle: {
-    color: "#896bb8",
-    fontSize: 16,
-    fontWeight: "600",
+  //from https://github.com/vonovak/react-navigation-header-buttons/blob/master/src/ButtonsWrapper.tsx#L58C3-L67C5
+  text: {
+    ...Platform.select({
+      android: {
+        color: "#896bb8",
+        fontFamily: "sans-serif-medium",
+        fontSize: 14,
+        textTransform: "uppercase",
+      },
+      default: {
+        color: "#896bb8",
+        fontSize: 17,
+        textTransform: "capitalize",
+      },
+    }),
+  },
+  extraEdgeMarginOnRight: {
+    ...Platform.select({
+      android: {
+        marginRight: 14,
+      },
+      default: {
+        marginRight: 15,
+      },
+    }),
```

Comment:
I took those settings from https://github.com/vonovak/react-navigation-header-buttons library.

I think they're thought through to achieve closest-to-native style. At the same time, I didn't want to introduce another dependency - at least for now, as we can without big effort have our own widget for header actions (`ToolbarActionButton`)